rest-server/server/src/main/resources/reference.conf

```diff
@@ -22,3 +22,11 @@ auth {
   client-id = "harness"
   client-secret = "mwjmo4fwC9qFW081gm5vwsmAC4Z2747XEU8eeV6dzUzjW"
 }
+
+elasticsearch {
+  uri = "http://localhost:9200"
+  auth {
+    username = ""
+    password = ""
+  }
```

Comment:
Auth should be disabled or using the same conf in harness-env.sh, the env, of fallback to the def here. Like Mongo.

With the exception of anything Spark needs, which should be derived from things known inside Harness already if possible. So if the SparkContext needs auth info it should be found from the `harness-env.sh` settings or in the env, ...